public/js/annotator-madison.js

```diff
@@ -280,15 +279,10 @@ $.extend(Annotator.Plugin.Madison.prototype, new Annotator.Plugin(), {
   addNoteLink: function (field, annotation) {
 
     //Add link to annotation
-    var annotationLink = $('<a></a>').attr('href', this.options.path + '#' + annotation.link).text('Copy Annotation Link').addClass('annotation-permalink');
-    var noteLink = $('<div class="annotation-link"></div>');
     var linkPath = this.options.origin + this.options.path + '#' + annotation.link;
+    var annotationLink = $('<a></a>').attr('href', linkPath).text('Permanent link to this note').addClass('annotation-permalink');
```

Comment:
Is there a way for this to use the `ng-translate` stuff instead of having the plain text? Are we concerned about that?